### PR TITLE
Move to button

### PR DESCRIPTION
**Pull request by ghost**
Ready to evaluate.

### CODE REVIEW

File: src/projects/diff/highlight.html [hunk 1]
```diff
@@ -317,13 +321,15 @@
...
       },
-
       flash: function() {
         this.toggleClass('flash', true);
         setTimeout(function() {
           this.toggleClass('flash', false);
           this.fire('stopped-flashing');
         }.bind(this), 800);
+      },
+      _promptToMove: function(sender) {
```
Comment (txsmith) [inline src/projects/diff/highlight.html:331]: This function argument is unused and should be removed.

File: src/projects/diff/diff.html [hunk 2]
```diff
@@ -113,6 +113,11 @@
       on-confirm="_addGroup"
     ></input-dialog>
 
+    <move-dialog id="moveGroupDialog"
+      action-title="Move commit to another group"
```
Comment (txsmith) [inline src/projects/diff/diff.html:117]: The word `commit` is not really correct in this context. The user moves a small part of a commit, which is formally called `hunk` (or just `change`).

**Review by txsmith — CHANGES_REQUESTED** [review]
Functionally this works perfectly :+1:
I do have some requests however:
- The button is overlay-ed by `.drag-handle` from `group.html`. This makes it hard to click the button. Making the drag-handle smaller would fix this I believe.
- In my opinion it would be better to have a `paper-menu-button` instead of a `paper-button`. This saves the need for an extra dialog.
- Please run `gulp lint` and fix the linter feedback wherever you think it is appropriate. I still see some inconsistencies for example in white-space and an unused argument.

**Comment by txsmith** [comment]
@eanker any thoughts?

**Comment by ghost** [comment]
@txsmith: Regarding the overlay and the choice of paper-button: I placed the button next to the "Approve this change" button, and it has the exact same type and css.

Also, regarding the choice paper-menu-button vs (paper-button + dialog): The reasoning behind the latter is that it makes sure that there is only one list of groups placed on the DOM (as contained by the dialog.) Had I gone for a paper-menu-button, every hunk would have to have its own list of groups, that would also have to be updated dynamically (as the list changes.)

I'm working on the rest of your comments right now.

File: src/projects/diff/diff.html (hunk 2 above)
Comment (ghost) [reply src/projects/diff/diff.html:117]: Done!

File: src/projects/diff/highlight.html (hunk 1 above)
Comment (ghost) [reply src/projects/diff/highlight.html:331]: Done!

**Comment by ghost** [comment]
I've run the linter and I've fixed all the changes in the files I've touched. If you like, I can fix the rest as well. Just let me know.

**Comment by txsmith** [comment]
Ok, we'll leave the dailog as it is.
For the buttons, I see that the classes are equal, but the `group` element dynamically creates an overlay element that obstructs the move button. It seems on a closer look that  it also obstructs the approve button. We'll extract an issue for this.

See this screenshot for clarification:
screenshot from 2016-10-02 20-57-11

**Comment by txsmith** [comment]
Extracted #376

**Comment by ghost** [comment]
I know what you mean. I think, though, that if you make the approve button visible you'll see the same thing happen to it as well. It will appear to the right of the move-to button. If you agree that the dialog (vs the paper-menu-button) is a good choice, then this is really a pre-existing issue. (perhaps add to the backlog? I can pick it up next.)

**Comment by txsmith** [comment]
Looks good!

**Pull request merged**